resources/carrier/zh/852.txt

```diff
@@ -0,0 +1,283 @@
+# Copyright (C) 2016 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Translations of en/852.txt from each carrier's wikipedia page and in
+# some cases the carrier's own home page.
+
+852510|中国移动
+852511|新移动通讯
+852512|中国移动
+852513|中国移动
+852514|香港移动通讯
+8525149|Truphone
+852516|中国移动
+852517|中国移动
+852518|香港移动通讯
+852519|3
```

Comment:
It would look somewhat strange to use a full-width "3" here, I think. The full-width slash also looks a bit weird. So I wonder if it's really common to make these substitutions?